roles/edpm_nodes_validation/tasks/main.yml

```diff
@@ -64,11 +64,26 @@
     - edpm_nodes_validation_validate_controllers_icmp|bool
     - edpm_nodes_validation_ping_test_ips | length > 0
 
-- name: Verify the configured FQDN vs /etc/hosts
+- name: Verify a FQDN is set block
+  when:
+    - edpm_nodes_validation_check_for_fqdn|bool
+  block:
+    - name: Run hostname
+      ansible.builtin.command: hostname
```

Comment:
I think you've to check `hostname -f` here which will return the fqdn .. This will fail when either hostname is not fqdn or there is no search domain in resolv.conf to attach to the shortname and resolv with dns configured.  Both pre-provisioned/baremetal jobs are failing in this PR because of that looks like.

Reply:
Done.